docs/patching-pega-deployment.md

```diff
@@ -0,0 +1,146 @@
+# Patching Pega Platform in your deployment
+
+After you deploy Pega Platform™ on your kubernetes environment, the Pega-provided Docker images support applying a zero-downtime patch to your Pega software. These procedures are written for any level of user, from a system administrator to a development engineer who wants to use helm charts and Pega Docker images to patch the Pega software have deployed in any supported kubernetes environment.
+
+Useful links to Pega software patching information:
+
+- [Pega software maintenance and extended support policy](https://community.pega.com/knowledgebase/articles/keeping-current-pega/85/pega-software-maintenance-and-extended-support-policy)
+- [Pega Infinity patch calendar](https://community.pega.com/knowledgebase/articles/keeping-current-pega/pega-infinity-patch-calendar)
+- [Pega Infinity patch frequently asked questions](https://community.pega.com/knowledgebase/articles/keeping-current-pega/85/pega-infinity-patch-frequently-asked-questions)
+
+## Deployment process overview
+
+Pega supports client-managed cloud clients applying patches for releases 8.4 and later using a zero-time patch process to apply the latest cumulative bundle of bug and security fixes since the last minor release. For the latest Pega Community articles, see [About client managed cloud](https://community.pega.com/knowledgebase/articles/client-managed-cloud/85/about-client-managed-cloud).
+
+The Pega zero-downtime patch process uses the out-of-place patch process so you and your customers can continue working in your application while you patch your system. Pega zero-downtime patch scripts use a temporary data schema and the patch migration script moves the rules between the appropriate schema and then performs the required rolling reboot of your deployment cluster. For a detailed overview of the process, see [Applying a patch without downtime](https://community.pega.com/knowledgebase/articles/keeping-current-pega/85/applying-patch-without-downtime).
```

Comment:
Temporary data schema changes are not merged to master yet. So will avoid talking about it in the above section.

Reply:
updated to remove specifics of temp data schemas: Pega zero-downtime patch scripts use a temporary schema and the patch migration script moves the rules between the schemas and then performs the required rolling reboot of your deployment cluster.